.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+<!-- Thanks so much for your PR, your contribution is appreciated! -->
+
+- [ ] Description explains the issue / use-case resolved, and auto-closes the related issue(s) (google it).
+- [ ] PR has tests / docs demo, and is linted.
+- [ ] Commit and PR titles begin with [ComponentName], and are in imperative form: "[Component] Fix leaky abstraction".
```

Comment:
Also, it's only necessary to follow that for the first one, as the rest will get squashed, and you can always `reword` when you squash, or `--amend` after.